rook/toolbox-deploy.yaml

```diff
@@ -17,10 +18,15 @@ spec:
       dnsPolicy: ClusterFirstWithHostNet
       containers:
         - name: rook-ceph-tools
-          image: rook/ceph:v1.5.3
-          command: ["/tini"]
-          args: ["-g", "--", "/usr/local/bin/toolbox.sh"]
+          image: rook/ceph:master
+          command: ["/bin/bash"]
```

Comment:
Any reason to this change?

Reply:
CrashLoop. :smile: The newer images do not include `/tini`. We could use an older version of `rook/ceph` in the toolbox, but since this is not on the "critical path" rather it's for debugging, I think that giving the operator the latest and greatest image is nicer.